src/Compilers/CSharp/Portable/Errors/MessageID.cs

```diff
@@ -149,6 +151,11 @@ internal static string RequiredFeature(this MessageID feature)
         {
```

Comment:
@khyperia agree. This PR is against future though which is why I commented about adding it here.